nixos/modules/config/no-x-libs.nix

```diff
@@ -34,6 +34,7 @@ with lib;
       ffmpeg_5 = super.ffmpeg_5.override { ffmpegVariant = "headless"; };
       # dep of graphviz, libXpm is optional for Xpm support
       gd = super.gd.override { withXorg = false; };
+      ghostscript = super.ghostscript.override { cupsSupport = false; x11Support = false; };
```

Comment:
Why no cups?

Reply:
I think I mainly based it on this comment https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/ghostscript/default.nix#L26-L27

and I also didn't need it in my use case. 

Not sure where to draw the line tbh.

Reply:
So it means without x11, cups adds a lot?

Reply:
Yes, significantly. 

```
 ➜ command nix-build --expr 'with import ./. { }; (ghostscript.override { cupsSupport = true; x11Support = false; })' --out result-with-cups
 ➜ command nix-build --expr 'with import ./. { }; (ghostscript.override { cupsSupport = false; x11Support = false; })' --out-link result-without-cups
 ➜ nix path-info -Sh ./result-with-cups
/nix/store/xhylpmfkyskcn7ql3gvya20xvcdwag42-ghostscript-10.02.1  195.0M
 ➜ nix path-info -Sh ./result-without-cups
/nix/store/6x3bd72jrlr47ljnf6yfxn9im9zhq9ka-ghostscript-10.02.1   99.2M
```